examples/objectdetection/src/CameraStream.tsx

```diff
@@ -37,6 +38,8 @@ interface Detection {
 type Props = BottomTabScreenProps<RootTabParamList, "CameraStream">;
 
 export const CameraStream: React.FC<Props> = () => {
+  const { settings } = useSettings();
+  const debouncedSettings = useDebounce(settings, 500);
```

Comment:
why?

Reply:
It takes long enough to tear down and spin up a new objectdetection object that it shows user visible delay when dragging the slider.  Also, it seems wrong to do that multiple times when someone is just dragging a slider.  The other way to fix this would be to make settings modal.  I'm happy to do that - this would probably involve going back to your idea of the camear screen being the "home" and using a stack navigator.  I'd rather not do that with this PR unless you feel strongly...